Gaming The Syllabus/Assets/Scripts/Combat/CombatManager.cs

```diff
@@ -109,7 +109,7 @@ public static CombatUnit[] GetTargetsByType(TargetType targetType)
     private static CombatUnit GetRandomLivingEnemy()
     {
         List<CombatUnit> livingEnemies = allCombatants.FindAll((enemy) => enemy.IsAlive);
-        return livingEnemies[Random.Range(0, livingEnemies.Count)];
+        return livingEnemies[Random.Range(1, livingEnemies.Count)];
```

Comment:
Do you remember why you needed to do this?

Reply:
There was a bug when testing out the combat against the final boss where when a player attacked the boss, it would occasionally also damage the player as well. The player is always the 0th index in the livingEnemies so the random.Range would sometimes return the 0th index lol I can see how this is confusing tho